tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
 - `.env.preprod`
 - `.env.prod`
  
- You can copy and rename the `.env.example` file in the `robot-tests` directory, replacing the variable values with those for that file's specific environment. The tests rely on these environment variables being set.
-
-
-## How do I run the tests?
+You can copy and rename the `.env.example` file in the `robot-tests` directory, replacing the variable values with those for that file's specific environment. The tests rely on these environment variables being set.
+
+Variables needed to run tests are as follows: 
+* `DATA_API_URL`: URL of the data API 
+* `PUBLIC_URL` : URL of the public frontend (protected with basic auth in non-production environments)
+* `ADMIN_URL`: (string) URL of the admin application
+* `ADMIN_EMAIL`: (string) Email of BAU test user
+* `ADMIN_PASSWORD` : (string) Password of BAU test user
+* `ANALYST_EMAIL`: (string) Email of Analyst test user 
+* `ANALYST_PASSWORD`: (string) Password of Analyst test user 
+* `PRERELEASE_EMAIL`: (string) Email of prerelease test user 
+* `PRERELEASE_PASSWORD`: (string) Password of prerelease test user 
+* `RELEASE_COMPLETE_WAIT`: (integer) The amount of time to wait for a release to publish 
+* `WAIT_MEDIUM`: (integer) timeout/sleep variable used throughout the tests 
+* `WAIT_LONG`: (integer) timeout/sleep variable used throughout the tests
+* `FAIL_TEST_SUITES_FAST`: (integer: 0 to 1) Fails test suite after any child test case fails 
+
+
+## **How do I run the tests?**
 
 From the `tests/robot-tests` directory run:
 
 ```
 pipenv run python run_tests.py
 ```
 
-Further instructions on available options
-```
-pipenv run python run_tests.py -h
-```
-
-## Running tests on the pipeline
+This is a python based CLI tool which is responsible for running the UI tests. There are numerous options which are documented below however we typically stick to the following two options:
 
-After changing the tests and ensuring they pass locally, it is then recommended that you run your changes on the pipeline to ensure they also pass there. There is a build pipeline you can use to check any changes you've made will pass on the pipeline: 'UI Tests'. You can run this pipeline using the branch with your changes, and it will produce an artifact you can download with the test report, screenshots, etc.
+- `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment>`
 
-At the time of writing, the pipeline runs the tests against the Dev environment, and runs all tests. You can edit the pipeline if you wish to do something different.
+- If you have a slow machine or are testing against a slow environment it is advisable to run the tests with the following options: `pipenv run python run_tests.py -f tests/path/to/robot/test -i robot -b chrome -e <environment> --timeout 20 --implicit-wait 20`
```

Comment:
You mentioned a timeout of 20 when running against a slow environment, but isn't this actually less time than the default?  I think the default is 30 seconds, but I could be wrong.